@kiva/kv-components/vue/KvCommentsList.vue

```diff
@@ -10,7 +10,7 @@
 			:user-display-name="userDisplayName"
 			:user-public-id="userPublicId"
 			:is-liked="comment.is_liked"
-			class="tw-mb-2"
+			class="tw-mb-2 tw-not-prose"
```

Comment:
What does `not-prose` add here?

Reply:
What issue specifically?

Reply:
Adding this style to every paragraph element `<p>`.

```
.tw-prose :where(p):not(:where([class~="tw-not-prose"] *)) {
    margin-top: 2rem;
    margin-bottom: 2rem;
} 
```